### PR TITLE
Fix padding in the library loading buttons

### LINKED ISSUE #2306: Outline is cut off in the library dropdown buttons
Screenshot 2020-10-28 at 8 47 04 PM

chrome/macos

### PR DESCRIPTION
**Pull request by lipis**
Fixes #2306 

### Before 
Screenshot 2020-11-03 at 6 40 15 PM

### After

Screenshot 2020-11-03 at 6 40 56 PM

### CODE REVIEW

**Comment by lipis** [comment]
Not sure if there is a better way of handling it..

File: src/components/Stack.tsx [hunk 1]
```diff
@@ -9,6 +9,7 @@ type StackProps = {
   align?: "start" | "center" | "end" | "baseline";
   justifyContent?: "center" | "space-around" | "space-between";
   className?: string | boolean;
+  style?: object;
```
Comment (dominictwlee) [inline src/components/Stack.tsx:12]:
```suggestion
  style?: React.CSSProperties;
```
Might be better to narrow type this.

Comment (lipis) [reply src/components/Stack.tsx]: Thanks!

Comment (lipis) [reply src/components/Stack.tsx]: Typescript is complaining though :/

Comment (dominictwlee) [reply src/components/Stack.tsx]: umm odd, will have a look.

**Comment by dominictwlee** [comment]
You can handle the type error with module augmentation by creating a `css.d.ts` file(or name it whatever you want). After that, you can remove CSSProperties type cast.

```
declare module "csstype" {
  interface Properties {
    // Add a CSS Custom Property
    "--gap"?: number;
  }
}
```

https://github.com/frenic/csstype#what-should-i-do-when-i-get-type-errors

**Comment by dwelle** [comment]
Fixed. At first I tried to put it in our `global.d.ts` but it didn't work well with the `import "csstype";` (required) for some reason, so created `css.g.ts`.

**Comment by huanghaiyang** [comment]
Type '{ "--gap": number | undefined; justifyItems: "start" | "center" | "end" | "baseline" | undefined; justifyContent: "center" | "space-around" | "space-between" | undefined; }' is not assignable to type 'Properties<string | number, string & {}>'.
  Object literal may only specify known properties, and '"--gap"' does not exist in type 'Properties<string | number, string & {}>'.  TS2322

    47 |       className={clsx("Stack Stack_vertical", className)}
    48 |       style={{
  > 49 |         "--gap": gap,
       |         ^
    50 |         justifyItems: align,
    51 |         justifyContent,
    52 |       }}

**Comment by huanghaiyang** [comment]
`
Type '{ "--gap": number | undefined; justifyItems: "start" | "center" | "end" | "baseline" | undefined; justifyContent: "center" | "space-around" | "space-between" | undefined; }' is not assignable to type 'Properties<string | number, string & {}>'.
  Object literal may only specify known properties, and '"--gap"' does not exist in type 'Properties<string | number, string & {}>'.  TS2322

    47 |       className={clsx("Stack Stack_vertical", className)}
    48 |       style={{
  > 49 |         "--gap": gap,
       |         ^
    50 |         justifyItems: align,
    51 |         justifyContent,
    52 |       }}
`

**Pull request merged**